drag-drop-swipe-recyclerview/src/main/java/com/ernestoyaquello/dragdropswiperecyclerview/DragDropSwipeAdapter.kt

```diff
@@ -456,20 +460,23 @@ abstract class DragDropSwipeAdapter<T, U : DragDropSwipeAdapter.ViewHolder>(
     }
 
     private fun onDragStartedImpl(viewHolder: U) {
+        if (viewHolder.adapterPosition == NO_POSITION) return
         val item = dataSet[viewHolder.adapterPosition]
 
         viewHolder.isBeingDragged = true
         onDragStarted(item, viewHolder)
     }
 
     private fun onSwipeStartedImpl(viewHolder: U) {
+        if (viewHolder.adapterPosition == NO_POSITION) return
         val item = dataSet[viewHolder.adapterPosition]
 
         viewHolder.isBeingSwiped = true
```

Comment:
In case the adapter position is equal to `NO_POSITION`, then `viewHolder.isBeingDragged` never gets set back to `false` because this line is not reached.